addons/project/security/ir.model.access.csv

```diff
@@ -3,22 +3,23 @@ access_project_project,project.project,model_project_project,project.group_proje
 access_project_project_manager,project.project,model_project_project,project.group_project_manager,1,1,1,1
 access_project_project_stage,project.project_stage,model_project_project_stage,base.group_user,1,0,0,0
 access_project_project_stage_manager,project.project_stage.manager,model_project_project_stage,project.group_project_manager,1,1,1,1
-access_project_task_type_user,project.task.type.user,model_project_task_type,base.group_user,1,0,0,0
+access_project_task_type_user,project.task.type.user,model_project_task_type,base.group_user,1,1,1,1
```

Comment:
I agree it could be a bit strange but I think it should be better to override the access rights and rules to be sure we will not have any security issues when todo app is uninstalled. 🙂

Reply:
Ok, but then we accept that private tasks have different behaviors in Project depending of the activation of todo, right? For example a non-project user will be able to edit a task he's assigned to in the case where todo is activated but not when it's uninstalled.

Reply:
As discussed, I have change the code to implement this behavior ;)